### DISCUSSION TITLE
More Project Player Buttons

### DISCUSSION THREAD

**Discussion by CopperCopleer** [ScratchAddons/ScratchAddons#1918] [Ideas]
pls Add Slow Mode, True Fullscreen, Frame, and More

**Comment by Endermaster3690** [comment]
> Slow Mode

change the fps in the alt click 60fps player

> True Fullscreen, Frame, and More

idk about that

**Comment by Secret-chest** [comment]
My ideas:
- turbo mode
- mute
- replace fullscreen with true fullscreen, but add it as an option
- miniplayer (shows over the rest of the website)
- 60 fps
- etc etc.

**Reply by Endermaster3690** [reply]
we have turbo mode
we have mute
we have 60 fps
